features/polyExplorer/src/screens/entityDetails/entityDetails.css

```diff
@@ -254,29 +262,72 @@
     margin: 38px 16px 0 16px;
 }
 
+.featured-map-container {
+    margin-bottom: 20px;
+}
+
+.featured-map-container .separator-unit h2,
+.location-map .separator-unit h2 {
+    margin: 32px 16px;
+    font-family: "Jost Medium";
+    font-size: 20px;
```

Comment:
Are these two together often? 
Maybe a utility class then? Don't need to do it here either way

Reply:
A utility class would be a great idea. Or maybe adding those properties to an `h2` at some "superior level" element, since I've seen this combination repeated several times after an `h2`:

```
    font-family: "Jost Medium";
    font-size: 20px;
    text-align: center;
```